core/src/main/java/com/graphhopper/routing/weighting/GenericWeighting.java

```diff
@@ -48,40 +40,30 @@
     protected final long headingPenaltyMillis;
     protected final double maxSpeed;
     protected final DataFlagEncoder gEncoder;
-    protected final double[] speedArray;
+    protected final DataFlagEncoder.Result result;
     protected final int accessType;
-    protected final int eventuallAccessiblePenalty = 10;
+    protected final int eventualAccessiblePenalty = 10;
```

Comment:
I think naming this `uncertainAccessiblePenalty`or something like this would be better. Eventual is a false friend to the German eventuell. I think this was introduced by me some time ago :), but since I just saw it, we should fix it.